app/models/health/careplan.rb

```diff
@@ -232,6 +232,13 @@ def rn_just_approved?
       ncm_approval? && rn_approved_on.present? && rn_approved_on_changed?
     end
 
+    def renewed_for_cp2?
+      # Only PCTPs from the 12 months before CP2 can be renewed
+      return false if provider_signed_on.blank? || provider_signed_on < Health::PatientReferral::CP_2_REFERRAL_DATE - 12.months
```

Comment:
What happens if the provider_signed_on date is >= health::PatientReferral::CP_2_REFERRAL_DATE ?

Reply:
PCP signatures ended on that date -- so, any date after that is not a completed CP1 referral.